north/src/runtime/state.rs

```diff
@@ -283,27 +297,27 @@ impl State {
     }
 
     /// Install a npk from give path
-    pub async fn install(&mut self, npk: &Path) -> result::Result<(), Error> {
-        npk::install(self, npk)
-            .await
-            .map_err(Error::InstallationError)?;
-        Ok(())
-    }
+    // pub async fn install(&mut self, npk: &Path) -> result::Result<(), Error> {
+    //     npk::install(self, npk)
+    //         .await
+    //         .map_err(Error::InstallationError)?;
+    //     Ok(())
+    // }
 
     /// Remove and umount a specific app
-    pub async fn uninstall(&mut self, app: &Application) -> result::Result<(), Error> {
-        if app.process_context().is_none() {
-            info!("Removing {}", app);
-            npk::uninstall(app.container())
-                .await
-                .map_err(Error::UninstallationError)?;
-            self.applications.remove(&app.manifest().name);
-            Ok(())
-        } else {
-            warn!("Cannot uninstall running container {}", app);
-            Err(Error::ApplicationRunning(vec![app.manifest().name.clone()]))
-        }
-    }
+    // pub async fn uninstall(&mut self, app: &Application) -> result::Result<(), Error> {
+    //     if app.process_context().is_none() {
+    //         info!("Removing {}", app);
+    //         npk::uninstall(app.container())
+    //             .await
+    //             .map_err(Error::UninstallationError)?;
+    //         self.applications.remove(&app.manifest().name);
+    //         Ok(())
+    //     } else {
+    //         warn!("Cannot uninstall running container {}", app);
+    //         Err(Error::ApplicationRunning(vec![app.manifest().name.clone()]))
+    //     }
+    // }
```

Comment:
missing in commit message

Reply:
Yes. I think I commented it because the install/uninstall/update thing is a subject to be totally reworked.

Reply:
agreed...not part of the pr though